tests/defaults.pyi

```diff
@@ -54,6 +58,19 @@ def f19(x: object = "foo" + 4) -> None: ...  # Y011 Only simple default values a
 def f20(x: int = 5 + 5) -> None: ...  # Y011 Only simple default values allowed for typed arguments
 def f21(x: complex = 3j - 3j) -> None: ...  # Y011 Only simple default values allowed for typed arguments
 def f22(x: complex = -42.5j + 4.3j) -> None: ...  # Y011 Only simple default values allowed for typed arguments
+def f220(x=MyEnum.VAL) -> None: ...  # Y014 Only simple default values allowed for arguments
+def f221(x: MyEnum = MyEnum.VAL) -> None: ...
+def f222(x: MyEnum = MyEnum.val) -> None: ...
+def f223(x: MyEnum | int = MyEnum.VAL) -> None: ...
+def f224(x: int | MyEnum = MyEnum.VAL) -> None: ...
+def f225(x: int = MyEnum.VAL) -> None: ...  # Y011 Only simple default values allowed for typed arguments
```

Comment:
this would arguably be a false positive if `MyEnum` is an `IntEnum`.

Should we maybe just allow arbitrary `Attribute` expressions as default values? The list of specific attributes we allow as default values feels like it's getting longer and longer. It seems pretty clear at this point that `Attribute` expressions as default values are pretty useful in lots of situations in stub files. We can probably trust type checkers to complain at us if we use an attribute expression as a default value and it's too complicated for the type checker to infer what the type is meant to be.

Reply:
I'm fine with that. I don't recall exactly why we wanted to limit the possible default values in the first place.

Reply:
Mainly a concern about only wanting to allow what we actually needed, and being worried about suddenly "opening the floodgates", I think